### PR TITLE
Update test resources creation script to generate sha256 certificates

### PR DESCRIPTION
**Pull request by abhipsaMisra**
These changes are a part of `pewviews/net5`, however, since it might be a while before they're merged into `mater`, it is worth getting these changes in specifically.

### CODE REVIEW

File: e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep [hunk 1]
```diff
@@ -85,6 +92,13 @@ param OperationalInsightsName string {
   }
 }
 
+param OperationInsightsLocation string {
+  default: 'eastus'
```
Comment (drwill-ms) [inline e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep:96]: What other region options do we have? Anything closer to the west coast?

**Review by drwill-ms — APPROVED** [review]

File: e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep (hunk 1 above)
Comment (abhipsaMisra) [reply e2e/test/prerequisites/E2ETestsSetup/test-resources.bicep:96]: We do have support for west us region, I'll update to that.

**Review by azabbasi — APPROVED** [review]

**Review by vinagesh — APPROVED** [review]

**Pull request merged**